lib/core/src/persist/migrations.rs

```diff
@@ -214,5 +214,10 @@ pub(crate) fn current_migrations() -> Vec<&'static str> {
         ) STRICT;",
         "ALTER TABLE receive_swaps DROP COLUMN mrh_script_pubkey;",
         "ALTER TABLE payment_details ADD COLUMN lnurl_info_json TEXT;",
+        "
+        ALTER TABLE receive_swaps ADD COLUMN expiry_at INTEGER;
+        ALTER TABLE send_swaps ADD COLUMN expiry_at INTEGER;
+        ALTER TABLE chain_swaps ADD COLUMN expiry_at INTEGER;
```

Comment:
Do we need to persist the expiry_at? It seem we have a way to calculate the timeout already so perhaps just calculate it when querying the payment and set it on the payment object?

Reply:
My rational was that:
- I didn't want to pass the chain tip data to the persister list_payments / get_payment to form the Payment response. 
- I don't see the need to recalculate the expiry after expiration/completion, but it's good to have the expiry for failed payments.
- we have to deserialize each swap creation json (except chain swaps) to calculate the expiry,  so just trying to be optimal.
I'm fine not persisting the expiry if you think it makes it simpler @roeierez

Reply:
I think the arguments you provided make sense.